search/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from . import views
```

Comment:
Can you please import it as `search_views`? it's how I called the views in seekers and apartments and I think its more clear if we will call all of them the same way.
Other than that looking great!

Reply:
Done.